contracts/USDCPool.sol

```diff
@@ -105,7 +105,7 @@ contract USDCPool is
      * @nonce calls by CruizePutOptions to lock the premiums
      * @param amount Amount of premiums that should be locked
      */
-    function sendPremium() external override payable onlyOwner {
+    function sendPremium() external  payable onlyOwner {
```

Comment:
The space between "external" & "payable" has 1 extra space 😛. Sorry its my OCD

Reply:
ohk  done